app/models/notifier.rb

```diff
@@ -172,6 +172,17 @@ def changeset_comment_notification(comment, recipient)
     end
   end
 
+  def new_issue_notification(issue_id,recipient)
+    with_recipient_locale recipient do
+      @url = url_for(:host => SERVER_URL,
+                     :controller => "issues",
+                     :action => "show",
+                     :id => issue_id)
+      subject = I18n.t("notifier.new_issue_notification.subject")
+      mail :to => recipient.email, :subject => subject
```

Comment:
We really need a centralized notification API, so mails are sent from one place, and a user (and editors) could see all notifications.
(not a bug here)